en/docs/administer/key-managers/configure-global-key-manager.md

```diff
@@ -0,0 +1,124 @@
+
+6. [Create an API]({{base_path}}/design/create-api/create-rest-api/create-a-rest-api/) (Eg: SampleAPI).
+
+    [![Sample API Creation]({{base_path}}/assets/img/administer/global-keymanager/sample-api-creation.png)]({{base_path}}/assets/img/administer/global-keymanager/sample-api-creation.png)
+
+7. Go to **Portal Configurations** from the left menu and click **Subscriptions**, click the **Subscription Availability** dropdown, and select the desired subscription availability option. For this scenario, select **Available to all the tenants** and click Save.
+
+    !!! Note
+        The **Subscription Availability** option will only be displayed if there are tenants in your environment.
+
+    [![Relationship between the API's visibility and subscription availability]({{base_path}}/assets/img/learn/api-subscription-availability.png)]({{base_path}}/assets/img/learn/api-subscription-availability.png)
+
+8. [Deploy the API]({{base_path}}/deploy-and-publish/deploy-on-gateway/deploy-api/deploy-an-api/).
+
+9. Sign in to the WSO2 API Publisher (`https://<hostname>:9443/publisher`) using the new tenant’s (abc.com) admin credentials and repeat Steps 5 - 7.
+
+10. Sign in to the super tenant's Developer Portal using the super tenant’s admin credentials (`https://<hostname>:9443/devportal`).
+
+11. [Create an application]({{base_path}}/consume/manage-application/create-application/) (Ex: SampleApp).
+
+    [![Sample App Creation]({{base_path}}/assets/img/administer/global-keymanager/sample-app-creation.png)]({{base_path}}/assets/img/administer/global-keymanager/sample-app-creation.png)
+
+12. Click **Production Keys** and Select the Global Key Manager (IS-GKM) which was created from the list of key managers displayed.
+
+    [![Select Global Key Manager]({{base_path}}/assets/img/administer/global-keymanager/select-global-key-manager.png)]({{base_path}}/assets/img/administer/global-keymanager/select-global-key-manager.png)
+
+13. Click **GENERATE KEYS**.
+
+14. Click **GENERATE ACCESS TOKEN** to generate an application access token. Make sure to copy the generated JWT access token that appears so that you can use it in the future.
+
+15. Go to the Developer Portal landing page, select the SampleAPI and [Subscribe]({{base_path}}/consume/manage-subscription/subscribe-to-an-api/#subscribe-to-an-existing-application) to the SampleApp.
+
+    [![Subscribe SampleAPI to SampleApp]({{base_path}}/assets/img/administer/global-keymanager/subscribe-sample-api-to-sample-app.png)]({{base_path}}/assets/img/administer/global-keymanager/subscribe-sample-api-to-sample-app.png)
+
+16. [Try Out the API]({{base_path}}/consume/invoke-apis/invoke-apis-using-tools/invoke-an-api-using-the-integrated-api-console/) with the access token generated using the Global Key Manager.
+
+    A Successful response indicates that an API of the super tenant can be invoked using an access token generated for an application using the Global Key Manager.
```

Comment:
Shall we change 'S' in "Successful" to lower case?

Reply:
fixed